source/fundamentals/transactions.txt

```diff
@@ -163,86 +173,88 @@ order details. Suppose you organize the data updates as follows:
 
-The documents in the ``inventory`` collection in this example contain the
-following:
+The following documents are in the ``inventory`` collection:
 
 .. code-block:: json
 
-   [
-    { name: "sunblock", sku: 5432, qty: 85 },
-    { name: "beach towel", sku: 7865, qty: 41 }
-   ]
+   { item: "sunblock", sku: 5432, qty: 85 },
+   { item: "beach towel", sku: 7865, qty: 41 }
 
 The code examples also perform operations on the ``orders`` collection, but
-do not require any prior sample documents.
+this collection does not contain any data before the transaction is
+performed.
 
 The code examples use the ``cart`` and ``payment`` variables to represent
-a sample list of items purchased and the order payment details as follows:
+a sample list of items purchased and the order payment details. The
+following code describes the contents of the ``cart`` and ``payment`` variables:
 
 .. code-block:: javascript
 
    const cart = [
-     { name: 'sunblock', sku: 5432, qty: 1, price: 5.19 },
-     { name: 'beach towel', sku: 7865, qty: 2, price: 15.99 }
+     { item: 'sunblock', sku: 5432, qty: 1, price: 5.19 },
+     { item: 'beach towel', sku: 7865, qty: 2, price: 15.99 }
    ];
    const payment = { customer: 98765, total: 37.17 };
 
 .. important::
-
-   The examples in the following sections require that you create the
-   collections outside of the transaction or that you are using MongoDB 4.4 or
-   later. For more information on creating collections inside a transaction,
-   see the
-   :manual:`Create Collections and Indexes in a Transaction </core/transactions/#std-label-transactions-create-collections-indexes>`
-   server guide.
+   
+   If you are connected to a deployment running MongoDB Server version
+   4.4 or later, you can create collections within transactions.
+   Otherwise, you must create the collections outside of the
+   transaction. To learn more about creating collections inside a
+   transaction, see :manual:`Create Collections and Indexes in a
+   Transaction
+   </core/transactions/#std-label-transactions-create-collections-indexes>`
+   in the Server manual.
 
 .. _nodejs-transaction-core-api-example:
 
 Core API Implementation
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-The code example in this section demonstrates how you can use the Core API to
-run the multi-document payment transaction in a session. This function
-shows how you can perform the following:
+The code example in this section demonstrates how to use the Core API to
+perform the multi-document transaction in a session. This example
+shows how to perform the following actions:
 
 1. Start a session
-#. Start a transaction, specifying transaction options
-#. Perform data operations in the same session
-#. Commit a transaction, or cancel it if the driver encounters an error
-#. End a session
+#. Start a transaction with transaction options
+#. Perform data operations in the session
+#. Commit the transaction, or abort it if the driver encounters an error
```

Comment:
Issue:

[Style guide ](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/) on "abort":

```suggestion
#. Commit the transaction, or end it if the driver encounters an error
```

Reply:
Regardless of the nomenclature of `abortTransaction`, I think in all instances that describe the action it takes, we should use "ends" per the style guide.